projectgenerator/qgs_project_generator.py

```diff
@@ -28,6 +30,8 @@ def __init__(self, iface):
         self.iface = iface
         self.__generate_action = None
         self.__configure_action = None
+        if locale.getlocale() == (None, None):
+            locale.setlocale(locale.LC_ALL, '')
```

Comment:
Those lines set the locale to the user's preferred locale (see https://docs.python.org/3.0/library/locale.html#locale.LC_ALL). It turns out that on our Windows machines `getlocale()` gives us `(None, None)`. Only after we call `locale.setlocale(locale.LC_ALL, '')`, we are able to get a proper encoding to use as `decode()` parameter.